charts/k8s-service/templates/canarydeployment.yaml

```diff
@@ -63,6 +63,9 @@ We need this because certain sections are omitted if there are no volumes or env
     {{- fail printf "configMaps config has unknown type: %s" (index . "as") -}}
   {{- end -}}
 {{- end -}}
+{{- if gt (len .Values.persistentVolumes) 0 -}}
```

Comment:
i think you might just be able to do `if .Values.persistentVolumes`, but i'm not sure if they're following a convention of using `len` or not

Reply:
They're already using that as a convention for checking maps, so probably best to leave as-is.